### ISSUE TITLE
Cannot set 0 value in gradient color setting

### ISSUE THREAD

**Issue by i1befree** [metatron-app/metatron-discovery#59]
**Describe the bug**
Color setting 을 커스텀으로 바꾸고 스크린샷처럼 0의 값을 흰색으로 지정하려 하나 0을 지정할 수 없음.

**To Reproduce**

**Expected behavior**

**Screenshots**
<img width="1547" alt="gradient_error" src="https://user-images.githubusercontent.com/526622/44442874-a5bac080-a60f-11e8-9fc0-eb203d6319e6.png">

**Desktop (please complete the following information):**
http://52.231.184.135:8185/

**Additional context**

**Comment by kyungtaak** [comment]
@AnnieHwang 해당 이슈 확인 부탁드립니다. 현재 UI 컴포넌트에서 값 설정에 불편함있네요.

**Comment by ecoloy** [comment]
@kyungtaak @AnnieHwang 리포트 된 바와 같이 값 설정에 불편함이 있음을 인지하고 있습니다만, 현재는 echart 스펙의 한계 때문에 간단히 해결될 이슈가 아닌것으로 분석되었습니다. 그래서 내부 지라 1325번 이슈로 이미 등록을 해두었고 시간을 두고 방안을 찾아보기로 한 상태입니다.

**Comment by i1befree** [comment]
@ecoloy 시작값과 종료값을 명시할 수 없는 이슈인지요? UI 상에서 선택을 못하는 것이 문제인지 아니면 echart 에서 시작과 종료를 선정 못해서 생기는 문제인지 궁금합니다.

**Comment by ecoloy** [comment]
@i1befree 먼저 관련 echart 예제를 공유드립니다. http://gallery.echartsjs.com/editor.html?c=xBJYz2XJz 
이 예제를 보시면 visualMap 옵션의 inRange color 가 현재 echart로 그라데이션을 구현하고 있는 옵션입니다.

echart가 그라데이션을 구현하는 옵션의 구조가 단순히 색상값을 여러개 입력받아 동일한 간격으로만 배치해주고 있습니다. 즉, 특정 색상을 특정 위치에 있도록 사용자가 지정할수가 없고 data의 min/max 안에서 균등하게 색상의 위치를 자동 배치시켜버립니다. 
그래서 현재 메타트론에서의 그라데이션 구현은 10개정도의 색상을 일정한 간격으로 배치할수 있게 해둔 것입니다. 그러나 이러한 제약조건 내에서 최대한 원하는 위치에 색상을 두는 방안을 고려중인데 그 중 내부 지라 1325번 이슈에서 아이디어를 낸게 색상의 갯수를 더 늘려서 세분화한다음에 그라데이션이 생성되는 범위의 min/max를 정수로 떨어지게 조정 하면 특정 색상이 0, 10과 같은 곳에 위치할수 있지 않을까 하는 방안입니다.

**Comment by kyungtaak** [comment]
@AnnieHwang 그라데이이션 컴포넌트 교체안 까지 검토해주시면 감사 드리겠습니다. 커스텀으로 입력이 가능하지 않다면 의미가 없을듯합니다.

**Comment by ecoloy** [comment]
@kyungtaak @AnnieHwang 오해가 있는것 같아 부연설명을 드리자면, 색상패널에 사용한 그라데이션 UI 컴포넌트와는 무관한 이슈이고 echart의 옵션에 제약이 있기 때문에 발생하는 현상입니다. echart가 제공하는 그라데이션 옵션이 사용자가 원하는 정확한 위치를 지정할수 없기때문에 색상 패널의 그라데이션 UI 컴포넌트를 거기에 맞게 제약을 두고 동작하도록 한것입니다. 이 현상을 근본적으로 해결하려면 echart가 아닌 다른 차트 라이브러리를 사용하거나 echart 측에 그라데이션 옵션의 개선을 요청해야할 것으로 판단됩니다.

**Comment by kyungtaak** [comment]
@AnnieHwang  @ecoloy 네 이해했습니다. 답변 감사합니다. 해당 이슈는 3.0.3 릴리즈 이후 위에 언급해주신것 처럼 세분화후 조정하는 방향을 포함하여 차트 내에서 커스터마이징을 통해 해결할 수 있는 방안이 있는지 조금더 확인해보고 개발여부를 결정 했으면 합니다.
